pkg/unikontainers/unikernels/unikraft.go

```diff
@@ -47,7 +48,19 @@ type UnikraftVFS struct {
 }
 
 func (u *Unikraft) CommandString() (string, error) {
-	return fmt.Sprintf("%s %s %s %s %s -- %s", u.AppName,
+	hasEnv := false
```

Comment:
We could avoid intermediate variables and string concatenation inside a loop:

```var envVarString string
if len(u.Env) > 0 {
	envVarString = "env.vars=[ " + strings.Join(u.Env, " ") + " ]"
}
```

Reply:
Yes, you are correct, the above version is much more simple.

Reply:
I have pushed an updated version, according to your suggestion.